client/my-sites/patterns/components/header/style.scss

```diff
@@ -0,0 +1,74 @@
+
+@import "@automattic/components/src/styles/typography";
+@import "@wordpress/base-styles/breakpoints.scss";
+
+.patterns-header {
+	background: #1d2327;
+	padding: 0 26px;
+
+	.patterns-header__inner {
+		padding: 105px 0 64px 84px;
+		background: url(./images/background.svg) no-repeat right bottom;
+		color: var(--studio-white);
+
+		h1 {
+			font-family: $font-recoleta;
+			font-size: rem(70px);
+			line-height: 1;
+			letter-spacing: 0.2px;
+			width: 600px;
+			padding-bottom: 24px;
+		}
+
+		.patterns-header__description {
+			font-family: $font-sf-pro-display;
+			font-size: rem(18px);
+			letter-spacing: 0.4px;
+			line-height: 1.4;
+			width: 450px;
+			padding-bottom: 32px;
+		}
+
+		.patterns-header__search-input {
+			border-radius: 4px;
+			width: 450px;
+			height: 56px;
+			overflow: hidden;
+			margin-bottom: 0;
+
+			input[type="search"] {
+				font-family: $font-sf-pro-text;
+				letter-spacing: -0.32px;
+
+				&::placeholder {
+					color: #01283d;
+				}
+			}
+		}
```

Comment:
Not a blocker, but ideally, I think we should nest this directly under `.patterns-header`